linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/dashboard.js

```diff
@@ -0,0 +1,45 @@
+"use strict";
+
+/**
+ * Number of millis to wait between data updates.
+ */
+var UPDATE_INTERVAL = 1000;
+
+$.when(
+  $.get("/admin/metrics.json")
+).done(function(metricsJson) {
+
+  $(function() {
+    var dashboard = Dashboard();
+    dashboard.start(UPDATE_INTERVAL);
+  });
+});
```

Comment:
just confirming that we'll eventually seed the initial page load with this first call to metrics.json, rather than "fill in content" after a second call?